### ISSUE TITLE
Add YouTube link to How to file a disability claim link

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#51388]
## Issue Description

A VBA partner asked us to add a YouTube link on our [How to file a VA disability claim](https://www.va.gov/disability/how-to-file-claim/) page. 

YouTube link: https://www.youtube.com/watch?v=8ORYYo-Q2Jc

We'll need to determine where on the page this link should go.

Before we add the link, VBA needs to make edits to the captions, which Beth already provided them with in an email. They also need to confirm that it's 508 compliant. Once we hear back from Tanekwa that these changes have been made, we can publish the link. 

## Tasks
- [x] Get updated link with caption edits
- [x] Add link to Drupal page
- [x] Review
- [x] Copy edit (Megan)
- [x] Publish (Megan)
- [x] Notify Tanekwa that page is live

## Acceptance Criteria
- [x] YouTube link is live on the page

**Comment by RLHecht** [comment]
Update: These captions have been updated so we can add the link to the page.

**Comment by katherine-fung** [comment]
Hi @laurwill, the youtube video is a broad overview of the process, and goes through the different bullet points in the first section ("How do I prepare before starting my application?"). 

I'm wondering if we could put it in one of 2 places: 
- as a link in the intro as a preface to the process
- as a link below the 4 bullet points in the section "How do I prepare before starting my application?"

What do you think?

**Comment by laurwill** [comment]
Hi @katherine-fung , 
We usually avoid links in the intro, so let's place this under the bullets in "How do I prepare"
Here's a similar YouTube link from the claim exam page — could use this as a model:
<img width="609" alt="image" src="https://user-images.githubusercontent.com/104158104/213448617-d67723db-f354-4859-b96f-0850ca682419.png">

**Comment by katherine-fung** [comment]
@laurwill - want to take a look at this page next week before we pass to Megan? 

I added the Youtube link. Our team also talked about changing the style of the bullets and links in the "How do I prepare" section. 

@megzehn made some great edits to separate the bulleted copy and links. 

I also removed the phrase "fully developed claims" from one of the links, since we don't explain what they are. 

[If we want to keep the link text "Learn about fully developed claims," we could also add something in the bulleted text to explain. 

i.e. Gather any evidence (supporting documents) you’ll submit yourself when you file your VA disability claim. We encourage you to send in all your supporting documents along with your claim. If your claim includes all the evidence we need (also called a fully developed claim), we may be able to process it more quickly. 

Learn about fully developed claims]
